src/Namshi/Notificator/Notification/Email/EmailNotification.php

```diff
@@ -9,33 +9,38 @@
  */
 class EmailNotification extends Notification implements EmailNotificationInterface
 {
-    protected $recipientAddress;
-    
+    /**
+     * Recipient Email Addresses.
+     *
+     * @var array
+     */
+    protected $recipientAddresses = [];
+
     /**
      * Constructor.
      * 
-     * @param string $recipientAddress
+     * @param array $recipientAddress
      * @param array $parameters
      */
-    public function __construct($recipientAddress, array $parameters = array())
+    public function __construct(array $recipientAddresses, array $parameters = array())
     {
-        $this->setRecipientAddress($recipientAddress);
+        $this->setRecipientAddresses($recipientAddresses);
```

Comment:
let's access the instance variable directly and lets remove the setter. It seems that we are using it just in the constructors